tests/integration/utils/kafka_server.py

```diff
@@ -192,5 +192,5 @@ def configure_and_start_kafka(
         ),
     )
     env: Dict[bytes, bytes] = {}
-    proc = Popen(kafka_cmd, env=env)
+    proc = Popen(kafka_cmd, env=env)  # pylint: disable=consider-using-with
```

Comment:
I think we should really handle these better, it's a good find by pylint, but it requires some refactoring.

Now when looking closer I noticed they only have one call site each, so perhaps it makes sense to refactor now?

Reply:
I pushed a separate PR for this: https://github.com/aiven/karapace/pull/534